src/main/java/mate/controller/CreateCarController.java

```diff
@@ -0,0 +1,41 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.Collections;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/add")

```

Comment:
```suggestion
@WebServlet(urlPatterns = "/cars/create")
```
 you use create verb in controller's name and jsp's name, so use it in the URL too to keep to consistency, the same for other Create controllers

Reply:
it was originally "create" but after common mistakes it became "add"
fixed)